python-sdk/example_dags/example_load_file.py

```diff
@@ -365,4 +366,13 @@
     )
     # [END load_file_example_27]
 
+    # [START load_file_example_28]
+    aql.load_file(
```

Comment:
I think that now might be a good time to split up our load file example dag. This dag is by far the longest running in our cicd and I am somewhat worried about adding more to it versus splitting it in half. WDYT?

Reply:
https://github.com/astronomer/astro-sdk/issues/1803 will address this